cfme/tests/intelligence/test_reports_with_timelines.py

```diff
@@ -1,39 +1,121 @@
+]
 
-@pytest.mark.manual
-@pytest.mark.tier(2)
-@pytest.mark.parametrize('report', timelines_reports)
-def test_default_reports_with_timelines(report):
-    """
-    Test that a timeline widget is displayed for the default reports.
-    Note that since the default reports look at last week, to test, we modify the reports
+PATHS = [
+    ["Events", "Policy", "Policy Events for Last Week"],
+    ["Events", "Policy", "Policy Events for the Last 7 Days"],
+    ["Configuration Management", "Hosts", "Date brought under Management for Last Week"],
+    ["Events", "Operations", "Operations VMs Powered On/Off for Last Week"],
+]
 
-    Polarion:
-        assignee: jdupuy
-        initialEstimate: 1/5h
-        casecomponent: web_ui
-        caseimportance: medium
-        testSteps:
-            1. Add an infrastructure provider to ensure that an event for each report will occur
-            2. Navigate to Cloud Intel -> Reports
-            3. Modify one of the default reports to display events from today
-            4. Navigate to Cloud Intel -> Timelines
-            5. Click on the report you created
-            6. Verify that the timeline (with events) is properly displayed
+UPDATES = [
+    {
+        'filter': {'primary_filter': "fill_field(Policy Event : Activity Sample - Timestamp ("
+                                     "Day/Time), ""IS, This Week)"},
+    },
+    {'report_fields': ["Activity Sample - Timestamp (Day/Time)",
+                       "Miq Policy Sets : Date ""Created", "Miq Policy Sets : Date Updated",
+                       "Miq Policy Sets : Description", "Miq Policy Sets : EVM Unique ID (Guid)"],
+     'filter':
+        {'primary_filter': "fill_field(Policy Event : Activity Sample - Timestamp (Day/Time), "
+            "IS, This Week)"},
+     },
+    {'filter':
+        {'primary_filter': "fill_field(Host / Node : Date Created, IS, This Week)"}},
+    {'filter':
+        {'primary_filter': "fill_field(Event Stream.Host / Node : Total VMs, >, 0)"}},
+]
+
+
+@pytest.fixture(scope="function")
+def setup_for_reports():
+    """ wrapper function for _setup_for_reports, so that it runs during test
+
+    Returns:
+        unbound function object for calling during the test
     """
-    pass
+    def _setup_for_reports(request, appliance, provider, path, updates, vm_crud, register_event): # noqa
+        """ This function makes copies of the default reports so that we can test the timelines
+        on the CloudIntel->Timelines page
+
+        Args:
+            request: py.test funcarg request
+            appliance: IPAppliance object
+            provider: provider object
+            path: path to the report in the tree
+            updates: updates to the default report so that it will display events
+            vm_crud: vm for Policy events
+            register_event: function for registering events
+            args: kwargs for extra_steps
```

Comment:
Required:
If you like to take some `kwargs` better to take as argument for `_setup_for_reports`

Reply:
@digitronik oh the `args: kwargs for extra_steps` is not meant to be there, I will remove it.